src/bidiMapper/BidiParser.ts

```diff
@@ -28,63 +28,71 @@ export interface IBidiParser {
   // Browsing Context domain
   // keep-sorted start block=yes
   parseCaptureScreenshotParams(
-    params: object
+    params: unknown
```

Comment:
`object` -> `unknown` seems to be a downgrade. It doesn't compile with `object` anymore?

Reply:
It's an upgrade since in theory we don't know if an object is passed in or not.

Reply:
+1 to keep this change.